.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```

Comment:
[OPTIONAL] Speaking of tracking linter errors locally. If you wanted to debug what's causing your Wehint and Stylint Linters to fail locally to can do the following:

First, you can include the `.hintrc` and `.stylelintrc.json` config files which can be found in the provided [linter for HTML and CSS](https://github.com/microverseinc/linters-config) repository.

Then, initialize npm to create the `package.json` file. on the command line/terminal run:
```
npm init -y
```
This will install the `node_modules` directory; which is why it is important to include it in the `.gitignore` file. Do not worry about understanding this at the moment, you'll learn about it in the next projects.

Next,
Install [Webhint](https://webhint.io/):
```
npm install --save-dev hint@6.x
```

Install [Stylelint](https://stylelint.io/):
```
npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
```

After the installations your root directory will look like this if you are using VScode:
![image](https://user-images.githubusercontent.com/38088818/164058625-7586dc35-93b3-49b2-b0d0-31f898def757.png)

Now let's check for some linter errors:
For WebHint; which checks HTML errors run:
```
npx hint .
```
In your case, you are missing the `charset` and `viewport` meta elements. As well as the “lang” attribute to the “html”.
![image](https://user-images.githubusercontent.com/38088818/164059828-7a114b51-2715-47a7-8136-43755cedb20d.png)

Therefore, to solve this your HTML should look like this:
![image](https://user-images.githubusercontent.com/38088818/164060293-7c0d05ad-1235-4535-8a88-eb0c466e6db0.png)


For Stylelint; which checks CSS errors run:
```
npx stylelint "**/*.{css,scss}"
```
In your case, you have the following errors in your `main.css` file:
![image](https://user-images.githubusercontent.com/38088818/164060568-3a1e4adb-cfd6-4cf1-afc8-a189c53d0d76.png)

To solve this, you can run:
```
npx stylelint "**/*.{css,scss}" --fix
```
This will auto-fix the style errors or you can fix them manually, whatever you prefer.
You now have a linter error free project!!! 👏🏾